style.css

```diff
@@ -134,4 +168,160 @@
   .icon-3 {
     height: 20px;
   }
+
+  .works {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 5%;
+  }
+
+  h1 {
+    width: 90%;
+    height: 40%;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  h2 {
+    width: 90%;
+    height: 40%;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  .text {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #344563;
+    line-height: 24px;
+    width: 90%;
+    padding: 2% 0;
+    font-size: 15px;
+  }
+
+  .project {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  #color {
+    color: #7f8cff;
+  }
+
+  .social {
+    display: flex;
+    align-self: start;
+    justify-content: space-around;
+    width: 60%;
+    margin: 1% 2%;
+  }
+
+  .card {
+    display: grid;
+    grid-gap: 1.5%;
+    margin-top: 20%;
+    box-sizing: border-box;
+    padding-top: 5%;
+    padding-bottom: 10%;
+    width: 90%;
+    height: auto;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  .text-card {
+    padding-left: 8%;
+  }
+
+  .canopy {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 0;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-size: 13px;
+    padding-top: 0;
+    color: #7a869a;
+  }
+
+  .cano {
+    color: #344563;
+    list-style-type: none;
+  }
+
+  .can {
+    color: #344563;
+  }
+
+  .pic {
+    justify-self: center;
+  }
+
+  ul {
+    display: flex;
+    justify-content: space-between;
+    width: 68%;
+    margin: 4% 0;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  .tag {
+    background: #ebebff;
+    border-radius: 8px;
+    padding: 2%;
+  }
+
+  .button {
+    display: flex;
+    justify-content: center;
+    width: 40%;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    padding: 4% 4%;
+    transition: 0.5s;
+    cursor: pointer;
+    color: #396df2;
+  }
+
+  .button:active {
+    background: #54d222;
+    color: #fff;
+  }
+
+  .last-button:active {
+    background: #54d222;
+    color: #fff;
+  }
+
+  .button:hover {
+    background: #6070ff;
+    color: #fff;
+    transition: 0.5s;
+  }
```

Comment:
- [ ] Great job adding the button interaction styles to your project. They are only missing the `pressed state` styles so kindly include this as well for all your buttons :100: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/236203347-e5c1a5e5-ba8d-47a0-90f6-2398a3f353b3.png)